module_4/ex_4_6_5.py

```diff
@@ -0,0 +1,152 @@
+'''
+Consignas:
+    Escribir funciones que resuelvan los siguientes problemas:
+    a) Dado un año indicar si es bisiesto.
+    Nota: un año es bisiesto si es un número divisible por 4, pero no si es divisible por 100, excepto que también sea divisible por 400.
+    b) Dado un mes y un año, devolver la cantidad de días correspondientes.
+    c) Dada una fecha (día, mes, año), indicar si es válida o no.
+    d) Dada una fecha, indicar los días que faltan hasta fin de mes.
+    e) Dada una fecha, indicar los días que faltan hasta fin de año.
+    f) Dada una fecha, indicar la cantidad de días transcurridos en ese año hasta esa fecha.
+    g) Dadas dos fechas (día1, mes1, año1, día2, mes2, año2), indicar el tiempo transcurrido entre ambas, en años, meses y días.
+    Nota: en todos los casos, invocar las funciones escritas previamente cuando sea posible.
+'''
+DIC_MESES_DIAS = {0:0, 1:31, 2:28, 3:31, 4:30, 5:31, 6:30, 7:31, 8:31, 9:30, 10:31, 11:30, 12:31}
+
+def es_bisiesto(y):
+    '''
+        Función que retorna True o False si un año es o no bisiesto.
+    '''
+    if(y % 400 == 0) or (y % 4 == 0 and y % 100 != 0):
+        return True
+    else:
```

Comment:
ni los returns ni el else son necesarios. basicamente si me veo retornando estados de un boolean dentro de un if significa que podria retornar la condicion del if.

`return y % 400 == 0 or y % 4 == 0 and y % 100 != 0`